Editor/UI/EditorDrawingHelper.cs

```diff
@@ -43,17 +45,28 @@ public static float HeaderLineHeight
             }
         }
 
+
         /// <summary>
         /// Draw button which is similar to default "Add Component" Unity button.
         /// </summary>
-        public static bool DrawAddButton(ref Rect rect, string label)
+        public static Rect calculateAddButtonRect(ref Rect rect)
```

Comment:
```suggestion
        public static Rect CalculateAddButtonRect(ref Rect rect)
```
Methosds are capitalized